### PR TITLE
Default windows_arch to x86 until Omnibus 5

### PR DESCRIPTION
**Pull request by jaym**
We're going to do an omnibus release, and we'd like to not have to bump the major version

### CODE REVIEW

**Comment by ksubrama** [comment]
:+1:

**Comment by yzl** [comment]
:+1:

File: lib/omnibus/config.rb [hunk 1]
```diff
@@ -452,11 +452,15 @@ def reset!
...
     # @return [:x86, :x64]
     default(:windows_arch) do
-        if Ohai['kernel']['machine'] == 'x86_64'
-            :x64
-        else
-            :x86
+      if Ohai['kernel']['machine'] == 'x86_64'
+        Omnibus.logger.deprecated('Config') do
+          "windows_arch is defaulting to :x86. In Omnibus 5, it will " \
+          "default to :x86_64 if the machine architecture is x86_64. " \
```
Comment (schisamo) [inline lib/omnibus/config.rb]: I think you mean `default to :x64`.

**Comment by schisamo** [comment]
Modulo my small comment I am :+1:.

**Pull request merged**